### DISCUSSION TITLE
Status shipments?

### DISCUSSION THREAD

**Discussion by strubbi77** [john30/ebusd#554] [Q&A] [answered]
When will nee devices be available?

Best regards Thomas

**Comment by john30** [comment]
we don't know, the chips are impossible to grab anywhere for months

EDIT: as stated below:
we might have a chance to get adapters built this summer, but it is still vague

**Reply by kugan49** [reply]
Hello,
Thank you for the answer. Can you update us on the progress in this thread?

**Reply by Peboli** [reply]
> we don't know, the chips are impossible to grab anywhere for months

Hi John,
Could you please give me the list of components that you can't actually find.
I can have a look on my side.

Best regards,

Oliver

**Reply by john30** [reply]
the isolators are usually the bottleneck, see the [circuit diagram](https://adapter.ebusd.eu/img/smd-circuit.png)

**Reply by john30** [reply]
we might have a chance to get adapters built this summer, but it is still vague

**Reply by strubbi77** [reply]
Is it possible to get what you have? Want to use IT off spec for BSB/LPB Bus, so have probably to do some changes anyway. But the board would be very helpful

**Reply by john30** [reply]
what do you mean? if the parts are available, we'll issue production and that includes the PCB and SMD soldering, so right now we don't have anything

**Reply by M0nk3yOo** [reply]
Hi John, one short question, just to be sure: I stared a reservation last week und confirmed the mail afterwards. Will I get another information as soon as you're ready with payment data?
I'm not sure if I missed sth.


Thanks in advance!
André

**Reply by john30** [reply]
> Will I get another information as soon as you're ready with payment data? I'm not sure if I missed sth.

yes

**Reply by lvelasco-upc** [reply]
Hi,
any update on the availability?
Thanks!
Luis

**Reply by john30** [reply]
we're still waiting for the isolator to become available, so no update on the timeline

**Reply by frederikbove** [reply]
Thx for the update, I’m also waiting on one

**Comment by sja** [comment]
It would be nice to have an indication on the ordering/reservation website if there are enough orders for the next build batch. I reserved one and I'm not sure if it would help if I order two of them to reach the limit or if that is contra productive.

**Reply by john30** [reply]
right now it is just a matter of the parts, not of the reservations

**Comment by dk7jb** [comment]
Hello John
What components are you missing for the next production? Can we help you with your search?
Sometimes some semiconductors are only available for a short time and are then sold out again for a longer period of time.
Jörn

**Reply by Peboli** [reply]
Hello @dk7jb,

You can check this reply from John30 : [https://github.com/john30/ebusd/discussions/554#discussioncomment-2319724](https://github.com/john30/ebusd/discussions/554#discussioncomment-2319724)

**Reply by dk7jb** [reply]
If help with the search is desired, more detailed information is required. Because of the different footprints, I would then have to have an exact specification and the number of semiconductors I was looking for.

**Reply by Peboli** [reply]
As far as I know BOM is not accessible, but you can easily see the package here : [https://adapter.ebusd.eu/img/smd-usb.jpg](https://adapter.ebusd.eu/img/smd-usb.jpg)
They are no 36 packages version for the component, it should not be an issue to check ;)

**Reply by Peboli** [reply]
But I think it will be hard to find some of them via regular distributor. Take for example the ISOW7842DWE, on most of the usual distributor the next delivery date is not before beginning of 2023. The best chance to find one is to search at broker side, and still without warranty. And if you find some at a broker the price will probably be insane ...

**Reply by Peboli** [reply]
@john30 You can still check with this company, for example they pretend to have 10.000 pcs of ISOW7842DWE and you can request for a quote depending of the quantity you plan to produce : [https://www.asourcingelectronics.com/product/detail/store/5260069/isow7842dwe.html](url)

**Reply by john30** [reply]
I got an offer from them for 100 isolators at a price of over 100 EUR per piece. not a real option

**Reply by Peboli** [reply]
@john30  This confirm what I said in one of my previous post, if you find some stock at a broker they try to sell at ten times the price ...

**Comment by john30** [comment]
good news: we got info that the missing isolator is supposed to be available by end of August! :-)  
It will take a little longer for adapters to be ready though, as the board needs to be changed slightly. The changes are done already, but need to be thoroughly verified and tested of course and as such, we hope that adapters are going to be ready around
**end of September**

**Reply by bjoerns1983** [reply]
From the status on the Website i see that the next batch is already completely sold?
So no further chance to get an adapter any time soon?

**Reply by ksimuk** [reply]
the same question. I am also interested in the adapter and the timeline for the next batch. just want to decide to try to do it myself or wait for the ready version

**Reply by john30** [reply]
"any time soon" is close to impossible due to the still ongoing chip crisis. like many others, we can't do any magic here and we will be more than happy when adapters are available after waiting almost a year for the isolators. this is supposed to happen in just a couple of days now! finally, *sigh*

**Reply by skipper-hqst** [reply]
The end of September is coming, any news regarding availability and shipments?

**Reply by john30** [reply]
we're still waiting for the supplier to send the isolators, this is so crazy (and frustrating)

**Reply by sharp7** [reply]
Hello,
on January 24th I acknowledged the reservation (via the link in the reservation mail I got that day at 20:34). I understand that you don't know when you get the missing parts, but I would appreciate it very much if you send out an information mail to the lucky ones so that they know if everything is going well and they are in the current batch or if there went something wrong with the reservation and it has to be done again.

**Comment by canogretic** [comment]
Hi John, 
I made the reservation around November last year. I don't remember getting any confirmation or anything like that. But it could also be that I haven't seen that. 
Is there a way to find out if I will be included in the next shipment or if I have to reservate the adapter again?

Unless I have overlooked this, a request for payment has not yet been received.

Maybe you can tell me shortly and I can initialize the next steps to get an Adapter. 

Best regards

Alex

**Reply by Thomas1305** [reply]
Same here.
I never received an email as described in the ordering process.

**Reply by john30** [reply]
guys, you received an email as stated on the reservation page when you submitted the request. otherwise you misspelled your mail address or it ended up in your spam folder. only if you acknowledged the link in that mail, your reservation is valid. further mails will only be sent once adapters are available or about to be.

**Reply by hbarnard** [reply]
That's the same for me as well. I'm not whining, you're doing great work but useful to know the reservation status (valid, lost etc.), order in queue etc. Of course, given the energy crisis, everyone will be worrying about these, so don't want to create additional pressure.

**Reply by ksimuk** [reply]
Just to put these into realistic perspective about saving, ebusd is amazing (I am using my own adapter while waiting for John's), but they won't save you thousands of pounds (or whatever currency you are using), most of the boilers are drastically oversized for the property they are installed in. My boiler's minimum power is 7kW, and the maximum property demand for heating is around 5-6kW. there is no modulation possible at any outside temperature. I have the flow at 50C (the minimum temperature my radiators and underfloor heating can dissipate 7kW) and just cycle on/off every couple of hours with an on/off controller. Yes, I can set the flow to a lower temperature, but this will force the boiler to cycle about every 10 minutes as radiators can't match the 7kW minimum output. Knowing numbers like gas consumption, modulation, return, and flow temperatures are interesting to create graphs, but won't save you money.

**Reply by michapr** [reply]
@ksimuk 
Why you think that adapter cannot save money?
Using ebusd adapter you can set the target temperature (as sample to night temperature) and cycle on/off this every couple of hours - or depending on room temperature. In this case your boiler will not cycle. 
Or did I misunderstand your post? ;)

**Comment by hbarnard** [comment]
Thanks. yes mine's already at 50 too, had a 'discussion' with the plumber. But I'm look for increased control and data rather than huge savings...

---------
https://www.hughbarnard.org
Twitter: @hughbarnard

------- Original Message -------

**Reply by john30** [reply]
@ksimuk & @hbarnard please don't misuse this discussion for completely different topics. feel free to open another one instead

**Comment by jplitza** [comment]
Maybe it's obvious, but: You can open the link in the original e-mail – the one that you used to confirm the reservation – to see the current status of affairs.

And while every month the shipment of parts is re-scheduled for the end of next month (which obviously is nothing @john30 can do anything about), it at least shows you weren't forgotten. :wink:

**Reply by M0nk3yOo** [reply]
Nice, thanks for the hint!

**Reply by Peboli** [reply]
We are in 2022, not in 2023 ;)

**Reply by Peboli** [reply]
@GieBek Obviously ?? Do you mean Farnell has made a mistake in the date ? I don't think so.
Sorry to tell that It seems you are not aware of the worldwide component shortage.
One year delay is quite common ...

**Reply by john30** [reply]
in the link you mentioned it is October 2023, not 2022. Welcome to the real world ;)

**Reply by sharp7** [reply]
Thank you for the information about the link in the original mail. I didn't expect it to work after the reservation, so I didn't try that. Now I know, I'm still on the list... :-)

**Reply by john30** [reply]
kind of a hidden feature indeed :-) we enhanced it a bit in the last weeks to reveal estimated availability.

**Comment by Peboli** [comment]
@GieBek The factory lead time for this component is 103 weeks, yes you read it correctly, 103 weeks.
Check the expected delivery time on Mouser too
![isol_mouser](https://user-images.githubusercontent.com/16508054/196263234-d1a56748-c38e-4a83-89b1-e16695247569.png)

**Comment by cyberthom42** [comment]
Any chance that reservations that came in after July 24th will get into the current charge? My reservation was sent end of  August.

**Reply by john30** [reply]
rather unlikely, unless a lot of guys that did reserver an adapter before will step back from their reservation

**Comment by john30** [comment]
great news: we finally got the missing parts and production has already started!
we'll inform one after the other about availability and readiness via email.
please be patient, as it takes time to handle everything.

**Reply by Peboli** [reply]
Hi John, great news indeed !
If we order now do we be in this delivery batch ? In other word do you launch the production for more interface than the amount of pre-order ?

**Reply by ra-kal** [reply]
I'm also interested. Any chance you can build more units?

> Hi John, great news indeed ! If we order now do we be in this delivery batch ? In other word do you launch the production for more interface than the amount of pre-order ?

**Reply by omenek159** [reply]
Me too. I did my reservation on October 3rd, so it would be great to receive it before end of the year :)

**Reply by frederikbove** [reply]
If you read the comments of John carefully about he can not guarantee anything for orders after mid July.

**Reply by ra-kal** [reply]
I will wait for official John statement.

**Reply by Peboli** [reply]
> If you read the comments of John carefully about he can not guarantee anything for orders after mid July.

Yes, most probably because he did not had a vision a component quantities that will be available.
But don't make plan on the comet and let's wait for his answer ;)

**Reply by john30** [reply]
reservations since July 2022 probably won't have a chance to get it within the next months. for the next batch we're again facing the same issue of non-availability of the chips anywhere (or for a completely ridiculous price of 200 eur or more). you can only wait and hope that people who reserved before that don't want it any more.

**Reply by claib** [reply]
Hi there
Any news ?
THX for the update

**Reply by john30** [reply]
update with regard to what exactly? we're handling the reservations one by one right now which surprisingly takes some time

**Reply by Thomas1305** [reply]
What types of chips are missing?
Maybe i can ask my contact in Asia.

**Comment by claib** [comment]
thx !
i mean my reservation from March.

Am Do., 10. Nov. 2022 um 08:20 Uhr schrieb John ***@***.***>:

> update with regard to what exactly? we're handling the reservations one by
> one right now which surprisingly takes some time
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/john30/ebusd/discussions/554#discussioncomment-4104749>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACAIWJJ4HSTHMZDSQCXX7GTWHSO4NANCNFSM5PR6H72Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

*Christoph Laib*
*VIOS GmbH, Switzerland*
*Speicherstrasse 27*
*CH-9053 Teufen*

M: +41 79 600 62 32
E:  ***@***.*** ***@***.***>
W: www.vios.ch

**Reply by john30** [reply]
you should have received it by now

**Comment by gnoffer** [comment]
i got my adapter today. will start implementing it soon. thanks @john30 !

**Comment by M0nk3yOo** [comment]
I got mine last week and is working well, thanks @john30! ✌️BestAndréAm 26.01.2023 um 11:18 schrieb Tom 'notifications at github.com' ***@***.***>:﻿
i got my adapter today. will start implementing it soon. thanks @john30 !

**Comment by john30** [comment]
we are busy since December 2022 in designing new layouts and every time we have one far enough and convincing to be good for the purpose, the chips are not available again. really an exhausting exercise...
now we had 2 completely new and also rather different layouts for which we ordered prototypes that arrived last week. we're still evaluating and testing them and one of the designs (which was my favourite) can't be used as the isolator gets hot (50 °C) which is according to the specs (which we oversaw) but not usable like that of course.
the second design is also working well without temperature issues and it is compliant to eBUS specs of course, but the isolator availability just moved into uncertain future (was supposed to arrive next week) - again.
So we're doing another redesign (revision 12) for the 5th isolator we found suitable and will probably order a prototype for that one next week. Then we'll have to thoroughly test it again and then we can tell more.
I'm rather confident with this one as we picked an isolator with very good availability, but you never know...

**Comment by lvelasco-upc** [comment]
Hi John,
Yes, thank you. Now I have to find time to install it :)
Luis

**Comment by mainmind83** [comment]
It would be nice to be able to buy in an online store...

**Reply by tehplague** [reply]
You are able to buy in an online store. Except that - at least to my understanding - those adapters are pretty much manufactured just-in-time or with little time before.

**Comment by GregorB54321** [comment]
I'm also still patient since Sept 2022. :-) But would be veery glad to hear, that shipment will start soon...
I was looking for a schematic to understand the function/specs of the isolator, but did not find any. May it would be possible to design it discrete?

**Comment by hgokduman** [comment]
I've reserved my adapter end of December 2022 - quite late compared to others here. ;) When I click on the link in the confirmation mail, it says "Waiting for further reservations on next batch, estimated for end of Q1/2023" - does that mean the batch will be manufactured or delivered by then?

**Comment by john30** [comment, marked as answer]
further steps taken:

1. we finalized the design based on the findings of testing the latest prototype
2. we started a small batch for production to also prove this process and not face severe issues with that (as we saw recently: SMT assembly engineers silently picked a part with the wrong footprint)
3. we are making good progress with the new firmware and web application
4. we started documentation (sneak preview available here, but only in German for now: https://adapter.ebusd.eu/v5/ )

So, if all goes well and no further Corona infection comes in between, we should have a batch of the new & great & tiny & full-of-capabilities device in around 3 weeks :-)

**Reply by tehplague** [reply]
Is the pricing going to stay the same?

**Reply by john30** [reply]
no, it's going to be cheaper!

**Reply by dk7jb** [reply]
What have you improved compared to version 3.1?

**Reply by john30** [reply]
look at the docs (and use Google translate if needed), then you'll see

**Reply by Danit2** [reply]
the new adapter looks great.  can't wait to get this one. 

which is the better connection?  usb or wifi?  

is wifi fast enough?  

if the adapter is connected to the ebusd via usb, can the web server still be used?  

Many thanks for the great project.

**Reply by zarch1972** [reply]
Great news John. Will these upgraded devices cover existing reservations?  I've been on the waiting since since September 2022 and can't wait to grab one. Thank you.

**Reply by john30** [reply]
the final prototype is fine and production has started. it'll take a bit to get them though.

to your questions:
> which is the better connection? usb or wifi?

USB will always be a bit better due to less hassle with WIFI connectivity. But if WIFI signal is good enough, it should not be a problem.

> if the adapter is connected to the ebusd via usb, can the web server still be used?

yes. the new firmware allows to use both in parallel.

> Will these upgraded devices cover existing reservations?

yes. in fact the reservation link now reveals infos about that and can be used to acknowledge the new design before payment. it also shows the estimated availabilty date now.

**Reply by zarch1972** [reply]
Thanks for the update @john30 , that's great news.
I've just accepted the new design.

Just a note, i'm getting 404 errors exploring the english v5 documentation
https://adapter.ebusd.eu/v5/raspberrypi.en
https://adapter.ebusd.eu/v5/index.en

The links work when I add .html to the end?

**Comment by mainmind83** [comment]
In my case, after confirming the reservation, updated information about the reservation never arrives... if there are many ahead... estimated date...
